infrastructure/cdn-in-a-box/cache/uname

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -o errexit -o nounset
+get_rhel_version() {
+	local redhat_release=/etc/redhat-release
+	local default_version="${RHEL_VERSION:-8}"
+	if [ -e $redhat_release ]; then
+		releasever="$(rpm -q --qf '%{version}' -f $redhat_release)"
+		releasever="${releasever%%.*}"
+	else
+		releasever=${default_version}
+	fi;
+
+	echo "el${releasever}"
+}
+rhel_version="$(get_rhel_version)"
+kernel_version="3.10.0-862.11.6.${rhel_version}"
+architecture=x86_64
```

Comment:
Do we need to spoof the architecture? All running code will, in fact, be using the host's architecture, not necessarily `x86_64`.

Reply:
You're right, no need to spoof. Still removed the `uname` shim in 549aa07ce.